src/main/java/seedu/guest/MainApp.java

```diff
@@ -16,18 +16,18 @@
 import seedu.guest.logic.Logic;
 import seedu.guest.logic.LogicManager;
 import seedu.guest.model.GuestBook;
+import seedu.guest.model.GuestPrefs;
 import seedu.guest.model.Model;
 import seedu.guest.model.ModelManager;
 import seedu.guest.model.ReadOnlyGuestBook;
-import seedu.guest.model.ReadOnlyUserPrefs;
-import seedu.guest.model.UserPrefs;
+import seedu.guest.model.ReadOnlyGuestPrefs;
 import seedu.guest.model.util.SampleDataUtil;
 import seedu.guest.storage.GuestBookStorage;
+import seedu.guest.storage.GuestPrefsStorage;
 import seedu.guest.storage.JsonGuestBookStorage;
-import seedu.guest.storage.JsonUserPrefsStorage;
```

Comment:
Should we stick with JsonUserPrefsStorage instead of changing it to JsonGuestPrefStorage? The user should be hotel managers and not guest. Likewise for the rest of the import statements, should we keep them as ReadOnlyUserPrefs and UserPrefs?